### ISSUE TITLE
not able to run this 

### ISSUE THREAD

**Issue by suriyaJaay** [onehungrymind/angular-animations-site#1]
Hi ,

am facing below error in cmd window after giving ng serve. everything installed properly
```

ERROR in ./node_modules/@angular/material/esm5/stepper.es5.js
Module not found: Error: Can't resolve '@angular/cdk/a11y' in 'C:\Users\jayasuriya.m\suriya\my workarounds\unzippedFiles\angular-animations-site-master\angular-animations-site-master\node_modules\@angular\material\esm5'
ERROR in ./node_modules/@angular/material/esm5/snack-bar.es5.js
Module not found: Error: Can't resolve '@angular/cdk/a11y' in 'C:\Users\jayasuriya.m\suriya\my workarounds\unzippedFiles\angular-animations-site-master\angular-animations-site-master\node_modules\@angular\material\esm5'
ERROR in ./node_modules/@angular/material/esm5/autocomplete.es5.js
Module not found: Error: Can't resolve '@angular/cdk/a11y' in 'C:\Users\jayasuriya.m\suriya\my workarounds\unzippedFiles\angular-animations-site-master\angular-animations-site-master\node_modules\@angular\material\esm5'
ERROR in ./node_modules/@angular/material/esm5/button.es5.js
Module not found: Error: Can't resolve '@angular/cdk/a11y' in 'C:\Users\jayasuriya.m\suriya\my workarounds\unzippedFiles\angular-animations-site-master\angular-animations-site-master\node_modules\@angular\material\esm5'
ERROR in ./node_modules/@angular/material/esm5/button-toggle.es5.js
Module not found: Error: Can't resolve '@angular/cdk/a11y' in 'C:\Users\jayasuriya.m\suriya\my workarounds\unzippedFiles\angular-animations-site-master\angular-animations-site-master\node_modules\@angular\material\esm5'
ERROR in ./node_modules/@angular/material/esm5/tooltip.es5.js
Module not found: Error: Can't resolve '@angular/cdk/a11y' in 'C:\Users\jayasuriya.m\suriya\my workarounds\unzippedFiles\angular-animations-site-master\angular-animations-site-master\node_modules\@angular\material\esm5'
ERROR in ./node_modules/@angular/material/esm5/chips.es5.js
Module not found: Error: Can't resolve '@angular/cdk/a11y' in 'C:\Users\jayasuriya.m\suriya\my workarounds\unzippedFiles\angular-animations-site-master\angular-animations-site-master\node_modules\@angular\material\esm5'
ERROR in ./node_modules/@angular/material/esm5/slider.es5.js
Module not found: Error: Can't resolve '@angular/cdk/a11y' in 'C:\Users\jayasuriya.m\suriya\my workarounds\unzippedFiles\angular-animations-site-master\angular-animations-site-master\node_modules\@angular\material\esm5'
ERROR in ./node_modules/@angular/material/esm5/datepicker.es5.js
Module not found: Error: Can't resolve '@angular/cdk/a11y' in 'C:\Users\jayasuriya.m\suriya\my workarounds\unzippedFiles\angular-animations-site-master\angular-animations-site-master\node_modules\@angular\material\esm5'
ERROR in ./node_modules/@angular/material/esm5/slide-toggle.es5.js
Module not found: Error: Can't resolve '@angular/cdk/a11y' in 'C:\Users\jayasuriya.m\suriya\my workarounds\unzippedFiles\angular-animations-site-master\angular-animations-site-master\node_modules\@angular\material\esm5
typings/step-label.d.ts (9,30): Cannot find module '@angular/cdk/stepper'.
ERROR in C:/Users/jayasuriya.m/suriya/my workarounds/unzippedFiles/angular-animations-site-master/angular-animations-site-master/node_modules/@angular/material/sort/typings/sort-header.d.ts (9,30): Cannot find module '@angular/cdk/table'.
ERROR in C:/Users/jayasuriya.m/suriya/my workarounds/unzippedFiles/angular-animations-site-master/angular-animations-site-master/node_modules/@angular/material/autocomplete/typings/autocomplete-trigger.d.ts (8,32): Cannot find module '@angular/cdk/bidi'.
ERROR in C:/Users/jayasuriya.m/suriya/my workarounds/unzippedFiles/angular-animations-site-master/angular-animations-site-master/node_modules/@angular/material/autocomplete/typings/autocomplete.d.ts (10,44): Cannot find module '@angular/cdk/a11y'.
ERROR in C:/Users/jayasuriya.m/suriya/my workarounds/unzippedFiles/angular-animations-site-master/angular-animations-site-master/node_modules/@angular/material/core/typings/ripple/ripple-renderer.d.ts (9,26): Cannot find module '@angular/cdk/platform'.

webpack: Failed to compile.

'
```

**Comment by christake** [comment]
@suriyaJaay Inside your `package.json` change `"@angular/material": "^2.0.0-beta.3"` to  `"@angular/material": "2.0.0-beta.3"` removing the ` ^`, then remove your node_modules and run `npm install` again.

**Comment by christake** [comment]
@suriyaJaay This issue has been fixed by #2

**Comment by suriyaJaay** [comment]
@aklukas thanks for your quick response .. now its working awesome
